frontend/src/utils/mappings.js

```diff
@@ -56,9 +56,18 @@ export const getApiType = (type) => {
     case "additive":
     case "active_ingredient":
     case "non_active_ingredient":
-      return "other-ingredients"
+      return "other-ingredient"
     default:
-      return `${type.replace("_", "-")}s`
+      return `${type.replace("_", "-")}`
+  }
+}
+
```

Comment:
Micro remarque pas du tout obligatoire... est-ce qu'on ne le changerait pour ne pas avoir _Not_ dans la fonction ? (pour éviter des trucs genre _if not not_)

On pourrait l'appeller `getActivityReadonlyByType` par ex ?

Reply:
Yes, merci pour ce rewording (je trouvais ce nom nul mais je n'avais pas réussi à en trouve un autre) 2ce57a191fbb1971eb2bad58e41956000738d9dc